tests/test_main.py

```diff
@@ -412,6 +412,17 @@ def test_keys_list(with_sample):
         ]
 
 
+def test_get_unique_values(with_sample):
+    with TestClient(app) as client:
+        response = client.get("/values/color")
+        assert response.status_code == 200
+        data = sorted(response.json(), key=lambda d: d["v"])
+        assert data == [
+            {'v': 'green', 'product_count': 1},
+            {'v': 'red', 'product_count': 2}
+        ]
```

Comment:
Great that you added a test.

Could you eventually add a test with a non existing key ?

Reply:
Done + one test for each param